### ISSUE TITLE
Multiple domains

### ISSUE THREAD

**Issue by mgldan** [okigan/synology-porkbun#1]
Thanks for the script! I have a Synology RT2600ac and this fits the exact niche I'm in with Porkbun.

I actually have two Porkbun domains. Once I set up Porkbun as my DDNS provider in the GUI, I was unable to select it again for the second domain. I tried setting up another instance of Porkbun inside ddns_provider.conf (same lines, different name), but the Synology GUI just hung - the QuickConnect & DDNS tab doesn't ever populate with the DDNS fast tab.

Any ideas for supporting this case? Grateful for your time in sharing and publishing this script.

**Comment by okigan** [comment]
That’s a limitation of Synology: only one domain per provider (applies even to preconfigured providers). As a work around you can create separate providers. For example: porkbun1 and porkbun2 and configure corresponding domains.

**Comment by mgldan** [comment]
Thanks for the reply. Indeed, making two providers ("Porkbun" and "PorkbunSecondary") was the first thing I tried. But the QuickConnect & DDNS tab just hung when I did.

I was able to get it to work through trial and error. Here's what I added to my ddns_provider.conf:

```
[Porkbun]
        modulepath=/usr/syno/bin/ddns/porkbun.sh
        queryurl=Porkbun
[AlsoPorkbun]
        modulepath=/usr/syno/bin/ddns/porkbun.sh
        queryurl=AlsoPorkbun

```
I would have preferred to start both provider names with the string "Porkbun", but it wasn't happy that way.
